bin/generate-podspecs.sh

```diff
@@ -17,8 +17,8 @@ function warn_missing_tag_commit() {
 }
 
 # Change to the expected directory.
-cd "$( dirname "$0" )"
-cd ..
+pushd "$( dirname "$0" )" > /dev/null
+popd > /dev/null
```

Comment:
The way I'm reading this, we `cd`/`pushd` into `dirname $0`, and immediately after `cd`/`popd` out of it.

Is this necessary to get some other automation to kick in? Otherwise, what do you think of removing it?